### PR TITLE
#1 Adds what is a PO

### PR DESCRIPTION
**Pull request by Cleop**
#1 Adds what is a Product Owner

### CODE REVIEW

**Review by nelsonic — APPROVED** [review]
😍

File: readme.md [hunk 1]
```diff
@@ -1,3 +1,42 @@
...
+8. Attach a _business_ or _customer_ value to each story/task in the backlog to
+ensure that we are building features that are genuinely wanted/needed by users. 💸
+9. Have a clear picture of "competitors" and the "market" for similar
+products/service to avoid "re-inventing" something that already exists! (_i.e.
+  don't try to replace App/Site ABC, you are wasting your time! instead find
+  something they aren't doing or are doing poorly and focus on that!_) 🔍
+10. Be familiar with _existing_ UX/UI through personal use of other/existing
+Apps/services to know what can be done/re-used. ♻️
+11. Trust that your technical team is knowledgeable/skilful/motivated and does not
+need "baby sitting" ... 🙄
```
Comment (iteles) [inline readme.md]: 😆 Maybe put this in a more politically correct way that doesn't make it sound like we think all POs are a thorn in our sides? 🙈

File: readme.md [hunk 2]
```diff
@@ -1,3 +1,42 @@
...
+methodology. They must understand and communicate the vision of the product to
+the rest of the team.
+
+They are the liaison between customers and the development team. This means that
+they are responsible for working with stakeholders and user groups to determine
+their needs. Then through collaboration with the Scrum Master and developer team
+they prioritise and discuss these ideas moulding them into features to be built.
+
+**PO Responsibilities:**
+1. Owner and creator of the product backlog* :crown:
```
Comment (iteles) [inline readme.md:18]: We're already getting into terminology that new POs (our audience) won't understand (like 'Scrum master' on line 14 above and 'product backlog' here), which will make people switch off pretty early on. Any thoughts on how we can mitigate this?

Comment (iteles) [reply readme.md:18]: Ooooooooh, I see what you did here with the asterisk 👍 

Still a question for other things like `scrum master`, `user testing` and `UI`.

Not a blocker for this PR, but worth an issue or solution.

File: readme.md [hunk 3]
```diff
@@ -1,3 +1,42 @@
...
+3. Approving or critiquing development work for release. :white_check_mark:
+4. Clearly define, understand and _empathise_ with the end-user. 👩 💡
+5. Involving customers or users and getting their feedback to continuously shape
+the product development. :+1: :thumbsdown:
+6. Maintaining and _updating_ the backlog: adjusting the features and _priority_
+as needed after each sprint or iteration. :small_red_triangle: :small_red_triangle_down:
+7. Describing the user-persona(s) in as much detail as possible to make the
+"pain" they feel _real_ (_especially useful when the team is remote and don't
+often meet with the end-users_) 📝
+8. Attach a _business_ or _customer_ value to each story/task in the backlog to
```
Comment (iteles) [inline readme.md]: This point might make more sense right next to the adding of new features to the backlog for example, but I don't know if you have already put these in an order that makes sense to you @Cleop ?

Not a blocker.

**Review by iteles — CHANGES_REQUESTED** [review]
Loving the semantic line breaks!

Just requesting a change on the eye rolling, the other comments are for your consideration @Cleop

File: readme.md (hunk 2 above)
Comment (Cleop) [reply readme.md:18]: I've created #22 for this, nice suggestion @iteles

**Comment by Cleop** [comment]
Thanks for reviewing @nelsonic @iteles 🎉 

@iteles - I've made your suggested changes. I find the ordering of the bullets question a tricky one because they all sort of cross over and I can't seem to make them all work in an order that works for one another. I've tried to integrate your suggestion so hopefully it's a little better than before.

**Review by iteles — APPROVED** [review]
Ace, thanks for the changes @Cleop

**Pull request merged**